src/main/java/com/eternalcode/parcellockers/parcel/ParcelRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.eternalcode.parcellockers.parcel;
+
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+public interface ParcelRepository {
+
+    CompletableFuture<Void> update(Parcel parcel);
```

Comment:
nazywaj voidy jednym stylem, jeżeli robisz update to następny void też powinień być find